styles/index.css

```
@@ -0,0 +1,103 @@
+.container {
+    border: solid black;
+    display: grid;
+    gap: 15px;
+    height: 100vh;
+    width: 100vw;
+    grid-template-columns: (2, 1fr);
```

No () and no commas here. 

```suggestion
    grid-template-columns: 2 1fr;
```